### ISSUE TITLE
5: Merge dictionaries: add option to dictionary editor to save current filter.

### ISSUE THREAD

**Issue by sofa13** [Germanika/plover#23]

**Comment by sofa13** [comment]
@morinted @Germanika @edalp008 Hey, for the merge of saving current filter etc, where do you think we should add the action? I think the best place would be somewhere in the Editor window, but in that case Ted, I'll need you to design another icon!

![icon_option](https://cloud.githubusercontent.com/assets/9827185/20492578/3a938c5c-afe3-11e6-9baf-30dc19ed076d.png)

![menu_option](https://cloud.githubusercontent.com/assets/9827185/20492584/3f5570ca-afe3-11e6-9329-e364dc3f2321.png)

**Comment by iangermann** [comment]
@sofa13 Yeah I think that's a pretty logical place to put it. It could also be useful to let the user select a bunch of entries, then right-click to merge/save-as that selection.

**Comment by panathea** [comment]
Good point Ian. I'd say that if we want to put it in the menu bar, then the
dictionary editor should get its own menu bar instead of hitching a ride on
the main window's.

On Nov 22, 2016 12:42 PM, "Ian Germann" notifications@github.com wrote:

@sofa13 https://github.com/sofa13 Yeah I think that's a pretty logical
place to put it. It could also be useful to let the user select a bunch of
entries, then right-click to merge/save-as that selection.

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
https://github.com/Germanika/plover/issues/23#issuecomment-262311255, or mute
the thread

**Comment by sofa13** [comment]
@Germanika and @morinted Okay great, thanks for your input guys, I agree with the points you both made. I think I'll create a menu bar for the dictionary editor and put the action there, as well as include a right-click to merge/save-as that selection.
